### PR TITLE
Update Cross Room Runways in Tourian Pt.1

### PR DESCRIPTION
**Pull request by kjbranch**
This is for the runways and the adjacent runways. Come in / leave charged is not included.

### CODE REVIEW

File: region/tourian/main.json [hunk 1]
```diff
@@ -3737,54 +3813,6 @@
...
-                {
-                  "name": "Base",
-                  "notable": false,
-                  "requires": []
-                }
-              ],
-              "openEnd": 0
-            },
-            {
-              "name": "Diagonal Shot Runway - Tourian Escape Room 1 Right Door (to Mother Brain)",
```
Comment (osse101) [inline region/tourian/main.json:3754]: These runways would still be used for `canLeaveShinecharged` out the bottom door.

File: region/tourian/main.json [hunk 2]
```diff
@@ -3494,11 +3537,44 @@
...
+              "strats": [
+                {
+                  "name": "Leave with Runway",
+                  "notable": false,
+                  "requires": [],
+                  "exitCondition": {
+                    "leaveWithRunway": {
+                      "length": 5,
+                      "openEnd": 1
+                    }
```
Comment (osse101) [inline region/tourian/main.json:3573]: Pretty easy to get hit by a rinka while taking this runway.

File: region/tourian/main.json (hunk 1 above)
Comment (osse101) [reply region/tourian/main.json:3754]: I see now that shinecharges are meant for a separate PR

Comment (kjbranch) [reply region/tourian/main.json:3754]: I was actually debating about this and took it out. I felt like getting through was harder than getting the 21 tile short charge. But that might not be the case with some beams. I'll look into it more

Comment (osse101) [reply region/tourian/main.json:3754]: That may be the appropriate way to look at it.  It is the same way of doing the room as speedruns, but they would have hyper beam.  And even then I wouldn't expect them to know the movements until `VH`

**Pull request merged**